### PR TITLE
Made the game responsive

### LINKED ISSUE #29: make the game responsive
since we need a wider display to play this game. Do the following

1. make sure the user is in **landscape** mode. if not just show some alert or something to tell them to change to landscape mode
2. once the user is in landscape mode, make the game adapt to it

### PR DESCRIPTION
**Pull request by DNA5769**
Fixes #29

### CODE REVIEW

**Comment by DNA5769** [comment]
Can you review this @dev-lovedeep

File: index.js [hunk 1]
```diff
@@ -10,10 +14,13 @@ function toggleSound(){
 // Wrapping entire code in anonymous function and calling it, so that user doesn't have access to cardImageSrcs
 (() => {
     let level = window.prompt("Choose a level", "1/2/3");
+    if (level !== '1' && level !== '2' && level !== '3')
+        level = '1'; // Setting default level to 1
```
Comment (dev-lovedeep) [inline index.js:18]: bonus for this :balloon:

File: index.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// Suggesting user to switch to landscape mode if their currently in portrait mode
+if (screen.availHeight > screen.availWidth)
+    alert("Please switch to landscape mode for a better experience!");
```
Comment (dev-lovedeep) [inline index.js:3]: this will check only one, instead add an event listener

**Review by dev-lovedeep — APPROVED** [review]
make this small change, and then we can merge

**Comment by DNA5769** [comment]
Adding a event listener for resize and putting alerts in it, made the game very buggy.
So instead I decided to display the message through text in the website.

File: style.css [hunk 3]
```diff
@@ -203,3 +206,39 @@ header{
...
+#orientation--message {
+    position: absolute;
+    top: 0px;
+    left: 0px;
+    height: 100vh;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 8rem;
```
Comment (dev-lovedeep) [inline style.css:224]: this is way too big change it to something like 2 or 3rem

**Pull request merged**